### ISSUE TITLE
Migrate from react Chartbar to Nova ChartBar

### ISSUE THREAD

**Issue by xoco70** [coroo/nova-chartjs#53]
Hi, 

I'm migrating my React app to Laravel Nova. I already have my graph in React, and I am having a hard time to migrate.

Here is React component:

```javascript
<Bar
        data={data}
        options={{
            legend: {
                labels: {
                    fontColor: "#FFF",
                    fontSize: 18
                },
                fullWidth: true,
                position: "bottom",
                fontColor: "#FFF"
            },
            maintainAspectRatio: false,
            scales: {
                xAxes: [
                    {
                        stacked: true,
                        gridLines: {
                            display: false
                        },
                        scaleLabel: {
                            fontColor: "#FFF"
                        },
                        ticks: {
                            fontColor: "#FFF"
                        }
                    }
                ],
                yAxes: [
                    {
                        scaleLabel: {
                            display: true,
                            labelString: "kWh",
                            fontColor: "#FFF",
                            padding: 0
                        },
                        ticks: {
                            min: 0,
                            fontColor: "#FFF"
                        },
                        gridLines: {
                            color: "rgba(255, 255, 255, 1)"
                        },
                        stacked: true
                    }
                ]
            }
        }}
    />
```

I tried to convert `xAxes`  but I can't make it work:

```php
->options(
                    [
                        'xaxis' => [
                            // Afficher chaque heure au lieu de chaque demi heure
                            'categories' => $result->map(function ($model) {
                                return Carbon::parse($model->ts)->format('h:i');
                            }),
                        ],
                        
                        'legend' => [
                            'position' => 'bottom'
                        ],
                        'maintainAspectRatio' => 'false',
                        'scales' => [
                            'yAxes' => [
                                'scaleLabel' => [
                                    'display' => true,
                                    'labelString' => "kWh",
                                    'fontColor' => "#FFF",
                                    'padding' => 0
                                ],
                                'ticks' => [
                                    'min' => 0,
                                    'fontColor' => false
                                ],
                                'gridLines' => [
                                    'display' => false
                                ],
                                'stacked' => true,
                            ]
                        ]
                    ]),
```

 it doesn't display the labelString kWh 

What should be the logic to migrate ? I can't make the yaxis legend "kWh" appears, I could change background for each serie, but I couldn't find a way to put the chart background in blue, can't find a way to change height, etc.

In short, I'm having a hard time to configure my charts, where I was not expecting to. 

Are all the original properties available ? Any logic I could follow to make it smooth?

I'm using Nova 2.x with Laravel 6

**Comment by coroo** [comment]
Hi @xoco70 , 

Sorry for late response. Unfortunately, we strict some default code from chart-js it self. The example is for height. Since height in laravel nova will overload the default card for laravel nova, so we make it strict. 

For better understanding what we can do until now, please follow the documentation at https://coroo.github.io/nova-chartjs/

Again, sorry for the inconvenience.
